src/utils/converter.tsx

```diff
@@ -24,3 +26,68 @@ export const convertScheduleToReactNode = (schedule: Schedule): ReactNode => {
     </>
   );
 };
+
+export const convertScheduleToSummaryReactNode = (
+  schedule: Schedule
+): ReactNode => {
+  return (
+    <>
+      {schedule.day1 && (
+        <>
+          {NITFES_DATE_TEXT.DAY1}: {formatPeriod(schedule.day1)}
+          <br />
+        </>
+      )}
+      {schedule.day2 && (
+        <>
+          {NITFES_DATE_TEXT.DAY2}: {formatPeriod(schedule.day2)}
+          <br />
+        </>
+      )}
+    </>
+  );
+};
+export const convertProjectDataToSwiperCardProps = (
+  project: ProjectData
+): SwiperCardProps => {
+  const props: SwiperCardProps = {
+    title: project.name,
+    href: `/project/${project.link}/`,
+    imageUrl: `/62nd/project/${project.link}/brochure.webp`,
+    imageAlt: project.name,
+    projectTag: project.tags,
+    schedule: convertScheduleToSummaryReactNode(project.schedule),
+    place: project.place,
+  };
+
+  if (!project.swiperOption) return props; // オプションがなければそのまま返す
+
+  // オプションがある場合は上書き
+  if (project.swiperOption.title) props.title = project.swiperOption.title;
+  if (project.swiperOption.dayTag) {
+    props.day1 = project.swiperOption.dayTag.day1;
+    props.day2 = project.swiperOption.dayTag.day2;
+  }
+  if (project.swiperOption.projectTag)
+    props.projectTag = project.swiperOption.projectTag;
+  if (project.swiperOption.place) props.place = project.swiperOption.place;
+  if (project.swiperOption.schedule) {
+    props.schedule = (
+      <span>
+        {project.swiperOption.schedule.day1 && (
+          <>
+            {NITFES_DATE_TEXT.DAY1}: {project.swiperOption.schedule.day1}
+            <br />
+          </>
+        )}
+        {project.swiperOption.schedule.day2 && (
+          <>
+            {NITFES_DATE_TEXT.DAY2}: {project.swiperOption.schedule.day2}
+          </>
+        )}
+      </span>
+    );
+  }
```

Comment:
```suggestion
  const { title, dayTag, projectTag, place, schedule } = project.swiperOption;
  if (title) props.title = title;
  if (dayTag) {
    props.day1 = dayTag.day1;
    props.day2 = dayTag.day2;
  }
  if (projectTag) props.projectTag = projectTag;
  if (place) props.place = place;
  if (schedule) {
    props.schedule = (
      <span>
        {schedule.day1 && (
          <>
            {NITFES_DATE_TEXT.DAY1}: {schedule.day1}
            <br />
          </>
        )}
        {schedule.day2 && (
          <>
            {NITFES_DATE_TEXT.DAY2}: {schedule.day2}
          </>
        )}
      </span>
    );
  }
```
リーダブルか怪しいのでこうするともっと読みやすいのかなと、余力があればお願いします

Reply:
もし変更しなくていいや～って感じなら教えてくれればapproveします